src/test/scala/uk/gov/hmrc/ui/pages/ReportOneLoginComplaintThanksPage.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ui.pages
+
+object ReportOneLoginComplaintThanksPage extends BasePage {
+  override val url: String                    = wrapUrl("/test-only/report-one-login-complaint/thanks")
+  override val expectedPageTitle: String      = "We have received your One Login for Government complaint"
```

Comment:
at the moment this fails, because the title of the thank you page is not the same as the page heading

what I've put here doesn't quite meet guidance either I think https://design.tax.service.gov.uk/hmrc-design-patterns/page-title/ we'd need GOV.UK on the end I believe - so might also want to change this test as well as the service